### PR TITLE
todo_React_TS_23.11.2022

### PR DESCRIPTION
**Pull request by CheetahAS**

### CODE REVIEW

**Comment by CheetahAS** [comment]
I.T. thank you for the review!

File: src/components/CreateTodoFrom.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, {useState} from 'react';
+import '../styles/styles.css'
+import { ITodo } from '../types/types';
+
+
+interface CreateTodoFormProps {
+    addTodo: (todo:ITodo) => void
+}
+
+const CreateTodoForm = ({addTodo}:CreateTodoFormProps) => {
```
Comment (gewalttat) [inline src/components/CreateTodoFrom.tsx]: (here and below) should be as ComponentName: React.FC < IProps > ({props})

File: src/components/CreateTodoFrom.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    const [value, setValue] = useState<string>('');
```
Comment (gewalttat) [inline src/components/CreateTodoFrom.tsx]: pls provide better naming than just [value, setValue], f.e. [todoText, setTodoText]

File: src/components/CreateTodoFrom.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    return (
+        <div className="todo__new">
+            <input 
+                onChange={changeValue} 
+                value={value} 
+                id="new" 
+                type="text" 
+                placeholder="Add a new task inside"
+            />
+            <div
```
Comment (gewalttat) [inline src/components/CreateTodoFrom.tsx]: (minor) here and below use < button > with some css instead, its semantic better

File: src/App.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    setTodoCount((prev) => prev - 1);
+  };
+
+  return (
+    <div className="todo">
+      <div className="todo__content">
+            <h1 className="todo__title">All tasks</h1>
+                <CreateTodoForm addTodo={(todo) => addTodo(todo)}/>
+                <TodoCounter todoCount={todoCount}/> 
+                <TasksList arrForTodos={arrForTodos} deleteTodo={(id) => deleteTodo(id)}/> {/* How does it work????????? */}
```
Comment (gewalttat) [inline src/App.tsx]: it works in () => deleteTodo(todo.id)

File: src/components/Task.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+const Task = ({todo, deleteTodo}:TaskProps) => {
+    const [isChecked, setIsCheked] = useState<boolean>(false);
+    
+    const checkBoxHandler = () => {
+        setIsCheked(!isChecked);
+    };
+
+    const cls = isChecked ? "todo__task todo__task-completed" : "todo__task";
```
Comment (gewalttat) [inline src/components/Task.tsx:17]: LGFM

**Review by gewalttat — COMMENTED** [review]
here is

**Pull request merged**